src/server/pkg/collection/collection_test.go

```diff
@@ -408,6 +398,79 @@ func TestTTLExtend(t *testing.T) {
 	require.True(t, actualTTL > TTL && actualTTL < LongerTTL, "actualTTL was %v", actualTTL)
 }
 
+func TestPagination(t *testing.T) {
```

Comment:
Should this be named TestIteration instead?

Reply:
Yeah that probably makes more sense, or maybe just `TestList` I called it `TestPagination` because the backend logic that it's testing is the pagination of etcd requests. But probably makes more sense to name it based on the interface it's using, rather than the implementation it's using.